libraries/AP_HAL_ChibiOS/hwdef/AIRLink/hwdef.dat

```diff
@@ -86,7 +86,7 @@ PG8 VDD_3V3_SENSORS4_EN OUTPUT HIGH
 # start peripheral power off, then enable after init
```

Comment:
probably need to remove this comment now

Reply:
Removed